src/otaclient/configs/_cfg_consts.py

```diff
@@ -57,9 +57,9 @@ def ACTIVE_ROOT(self) -> str:  # NOSONAR
     OTA_TMP_STORE = "/.ota-tmp"
     """For backward-compatibility"""
     OTA_RESOURCES_STORE = "/.ota-resources"
-    """tmp store for local copy, located at standby slot."""
     OTA_META_STORE = "/.ota-meta"
     OTA_META_STORE_BASE_FILE_TABLE = "/.ota-meta/base"
+    OTA_DOWNLOAD_DIR = "/.ota-download"
```

Comment:
Use a separated folder to hold in-complete downloading files or intermediate temp resources.
This will enable OTA download resume feature for new OTA image.